resources/indexer.py

```diff
@@ -339,53 +526,26 @@ def process_dir(path, site):
     dirs = []
     file_meta = []
 
-    # get directory's metadata
-    run_meta_xml = None
-    gaps_files = dict()  # gaps_files[<filename w/o extension>]
-    gcd_files = dict()  # gcd_files[<run id w/o leading zeros>]
-    for dir_entry in scan:
-        if "meta.xml" in dir_entry.name:  # Ex. level2_meta.xml
-            with open(dir_entry.path, 'r') as f:
-                run_meta_xml = xmltodict.parse(f.read())
-        if "_GapsTxt.tar" in dir_entry.name:  # Ex. Run00130484_GapsTxt.tar
-            with tarfile.open(dir_entry.path) as tar:
-                for tar_obj in tar:
-                    file = tar.extractfile(tar_obj)
-                    file_dict = yaml.load(file)
-                    # Ex. Level2_IC86.2017_data_Run00130484_Subrun00000000_00000188_gaps.txt
-                    name = tar_obj.name.split("_gaps.txt")[0]
-                    gaps_files[name] = file_dict
-        if "GCD" in dir_entry.name:  # Ex. Level2_IC86.2017_data_Run00130484_0101_71_375_GCD.i3.zst
-            run = _get_run_number(dir_entry)
-            gcd_files[str(run)] = dir_entry.path
+    manager = MetadataManager(path, site)
 
     # get files' metadata
     for dir_entry in scan:
         if dir_entry.is_symlink():
             continue
         elif dir_entry.is_dir():
+            logging.debug(f'Directory appended, {dir_entry.path}')
             dirs.append(dir_entry.path)
         elif dir_entry.is_file():
+            logging.debug(f'Gathering metadata for {dir_entry.name}...')
```

Comment:
Again, keeping the debug logging or removing it?

Reply:
I find it useful.